content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -9,36 +9,45 @@ This page describes authentication for Kubeflow Pipelines to GCP.
 ## Before you begin
 [Installation Options for Kubeflow Pipelines](/docs/pipelines/installation/overview/) introduces options to install Pipelines. Be aware that authentication support and cluster setup instructions will vary depending on the option you installed Kubeflow Pipelines with.
 
+## Compute Engine default service account
+
+This is good for trying out Kubeflow Pipelines, because it is easy to set up, but not secure.
+
+By default, your GKE nodes use [Compute Engine default service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account). If you allowed all scopes when creating the cluster,
```

Comment:
I think it'll be better to add a link or a picture here to demonstrate what `all scope` means. Because it's not literally called `all scope` on the cluster creation UI, it might cause confusion.

Reply:
Thanks! That's a good point. Let me adjust

Reply:
Adjusted, PTAL!

Reply:
LGTM. Thanks!